.snyk

```diff
@@ -0,0 +1,5 @@
+ignore:
+  MPL-2.0:
+    - reason: "Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license"
+
+
```

Comment:
What is the purpose for the unnecessary blank lines?

Reply:
@BlaineEXE  wrote:
> What is the purpose for the blank lines?

There is none. Added by accident and removed now. Thanks for pointing out!